index.html

```diff
@@ -0,0 +1,190 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>ICT4D: ICT4D for Development of Africa</title>
+  <link rel="stylesheet" href="./css/styles.css">
+  <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+</head>
+<body>
+  <div class="wrapper">
+    <header class="header1">
+      <ul class="little-header-list">
+        <li class="list-item">
+          <a href="#"> <i class="fa fa-facebook"></i> </a>
+        </li>
+        <li class="list-item">
+          <a href="#"> <i class="fa fa-twitter"></i> </a>
+        </li>
+        <li class="list-item">
+          <a href="#"> English </a>
+        </li>
+        <li class="list-item">
+          <a href="#"> My Page </a>
+        </li>
+        <li class="list-item">
+          <a href="#"> Logout </a>
+        </li>
+      </ul>
+
+    </header>
+    <header class="header2">
+      <div class="menu-bar-container">
+        <ul class="menu-bar">
+          <li>
+           <a href="#" class="hamburger">
```

Comment:
- { ] Kindly follow the given design. remove `logo` and desktop `nav` in the mobile view.

Your design
![image](https://user-images.githubusercontent.com/81932478/191732009-96b25388-66b6-4f7e-b067-08884ed2724f.png)

Given design
![image](https://user-images.githubusercontent.com/81932478/191732099-405f65db-fcc6-4401-b1fc-d3cf601cc595.png)